Frontend/src/Product.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
```

Comment:
dla mnie spoko tylko klika rzeczy:
 - z czego będzie korzystać do tworzenia komponentów UI ? Czy zostaje bootstrap, czy pójdziecie w ten react ui czy coś innego ?
 - można by się postarać w dalszej kolejności wydzielić to na podfoldery oraz jakieś generyczne kompoenty jeśli oczywiście jest taka sposobność np. w kilku miejscach jest lista elementów to można to dać jak coś reużwalnego.

Reply:
Na razie komponenty są z react-bootstrap, pewnie przerobię na MUI zgodnie z dyskusją na zajęciach
Co do wydzielenia generic komponentów - pewnie, zajmę się tym na etapie widoków i podstron

Reply:
UPDATE: pozmieniałam trochę na podfoldery i sensowniejsze komponenty